src/database/gravity-db.c

```diff
@@ -119,37 +124,37 @@ bool gravityDB_getTable(const unsigned char list)
 {
 	if(!gravity_database_avail)
 	{
-		logg("gravityDB_getTable(%d): Gravity database not available", list);
+		logg("gravityDB_getTable(%u): Gravity database not available", list);
+		return false;
+	}
+
+	// Checking for smaller than GRAVITY_LIST is omitted due to list being unsigned
```

Comment:
`GRAVITY_LIST` is now `GRAVITY_TABLE`

Reply:
Since this is only a minor comment inaccuracy, I will change it in another PR.